test/e2e-common.sh

```diff
@@ -35,7 +35,7 @@ function print_diagnostic_info() {
 
 function install_pipeline_crd() {
   echo ">> Deploying Tekton Pipelines"
-  kubectl apply --filename https://storage.googleapis.com/tekton-releases/latest/release.yaml || fail_test "Tekton pipeline installation failed"
+  kubectl apply --filename https://storage.googleapis.com/tekton-releases/v0.5.2/release.yaml || fail_test "Tekton pipeline installation failed"
```

Comment:
URL should probably be https://storage.googleapis.com/tekton-releases/previous/v0.5.2/release.yaml